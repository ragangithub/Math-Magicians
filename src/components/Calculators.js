import React from 'react';
import Display from './Displays';

const Calculators = () => (
  <div className="calculator">
    <h2 className="text">Let&apos;s do some Math!</h2>
    <Display />
  </div>
);

export default Calculators;
