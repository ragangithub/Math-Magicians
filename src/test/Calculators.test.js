import React from 'react';
import { render } from '@testing-library/react';
import Calculators from '../components/Calculators';

describe('Quote component', () => {
  test('renders correctly', () => {
    const component = render(<Calculators />);
    expect(component).toMatchSnapshot();
  });
});
