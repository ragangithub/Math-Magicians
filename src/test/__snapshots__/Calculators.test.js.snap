// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="calculator"
      >
        <h2
          class="text"
        >
          Let's do some Math!
        </h2>
        <div>
          <div
            class="display-container"
          >
            <div
              class="display display-2"
            >
              0
            </div>
          </div>
          <div
            class="buttons"
          >
            <button
              class="button"
              type="button"
            >
              AC
            </button>
            <button
              class="button"
              type="button"
            >
              +/-
            </button>
            <button
              class="button"
              type="button"
            >
              %
            </button>
            <button
              class="button operator"
              type="button"
            >
              /
            </button>
            <button
              class="button"
              type="button"
            >
              7
            </button>
            <button
              class="button"
              type="button"
            >
              8
            </button>
            <button
              class="button"
              type="button"
            >
              9
            </button>
            <button
              class="button operator"
              type="button"
            >
              X
            </button>
            <button
              class="button"
              type="button"
            >
              4
            </button>
            <button
              class="button"
              type="button"
            >
              5
            </button>
            <button
              class="button"
              type="button"
            >
              6
            </button>
            <button
              class="button operator"
              type="button"
            >
              -
            </button>
            <button
              class="button"
              type="button"
            >
              1
            </button>
            <button
              class="button"
              type="button"
            >
              2
            </button>
            <button
              class="button"
              type="button"
            >
              3
            </button>
            <button
              class="button operator"
              type="button"
            >
              +
            </button>
            <button
              class="button span"
              type="button"
            >
              0
            </button>
            <button
              class="button"
              type="button"
            >
              .
            </button>
            <button
              class="button operator"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="calculator"
    >
      <h2
        class="text"
      >
        Let's do some Math!
      </h2>
      <div>
        <div
          class="display-container"
        >
          <div
            class="display display-2"
          >
            0
          </div>
        </div>
        <div
          class="buttons"
        >
          <button
            class="button"
            type="button"
          >
            AC
          </button>
          <button
            class="button"
            type="button"
          >
            +/-
          </button>
          <button
            class="button"
            type="button"
          >
            %
          </button>
          <button
            class="button operator"
            type="button"
          >
            /
          </button>
          <button
            class="button"
            type="button"
          >
            7
          </button>
          <button
            class="button"
            type="button"
          >
            8
          </button>
          <button
            class="button"
            type="button"
          >
            9
          </button>
          <button
            class="button operator"
            type="button"
          >
            X
          </button>
          <button
            class="button"
            type="button"
          >
            4
          </button>
          <button
            class="button"
            type="button"
          >
            5
          </button>
          <button
            class="button"
            type="button"
          >
            6
          </button>
          <button
            class="button operator"
            type="button"
          >
            -
          </button>
          <button
            class="button"
            type="button"
          >
            1
          </button>
          <button
            class="button"
            type="button"
          >
            2
          </button>
          <button
            class="button"
            type="button"
          >
            3
          </button>
          <button
            class="button operator"
            type="button"
          >
            +
          </button>
          <button
            class="button span"
            type="button"
          >
            0
          </button>
          <button
            class="button"
            type="button"
          >
            .
          </button>
          <button
            class="button operator"
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
